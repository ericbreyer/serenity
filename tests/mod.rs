// #![cfg(test)]

// use insta::Settings;
// use test_case::test_case;

// #[test_case("test12.ser"; "test12")]
// #[test_case("test11.ser"; "test11")]
// #[test_case("test10.ser"; "test10")]
// #[test_case("test6.ser"; "test6")]
// #[test_case("test5.ser"; "test5")]
// #[test_case("test4.ser"; "test4")]
// #[test_case("test3.ser"; "test3")]
// #[test_case("test2.ser"; "test2")]
// pub fn test_file(file : &str) {

//     let mut out = Vec::new();
//     serenity::run_file(file, &mut out).unwrap();

//     let mut settings = Settings::clone_current();
//     settings.set_snapshot_suffix(format!("{file}"));
//     let _guard = settings.bind_to_scope();
//     insta::assert_snapshot!(std::str::from_utf8(&out).unwrap());
// }
