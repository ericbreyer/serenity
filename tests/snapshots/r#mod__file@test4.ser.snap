---
source: tests/mod.rs
expression: "std::str::from_utf8(&out).unwrap()"
---
global *0 global *40:0
0
0
0
-----
1
1
1
-----
2
1
1
-----
3
2
2
-----
4
3
3
-----
5
5
5
-----
6
8
8
-----
7
13
13
-----
8
21
21
-----
9
34
34
-----
10
55
55
-----
11
89
89
-----
12
144
144
-----
13
233
233
-----
14
377
377
-----
